### PR TITLE
Commerce module refactor

### PR DESCRIPTION
**Pull request by saulecabrera**
Here's the initial refactoring of the Commerce module according to the discussion in Issue #12. I've also used the second approach stated in the discussion to organize the files (e.g. ```lib/commerce.ex``` and ```lib/commerce/en.ex```) although it's not specified in the discussion which one is going to be used

### CODE REVIEW

File: lib/faker/commerce/en.ex [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  Returns a complete product name, based on product adjectives, product materials, product names
+  """
+  @spec product_name() :: String.t
+  def product_name do
+    product_name(:crypto.rand_uniform(0, 3))
+  end
+
+  defp product_name(0), do: "#{product_name_adjective} #{product_name_material} #{product_name_product}"
+  defp product_name(1), do: "#{product_name_adjective} #{product_name_product}"
+  defp product_name(3), do: "#{product_name_material} #{product_name_product}"
```
Comment (igas) [inline lib/faker/commerce/en.ex]: I think it should be `(2)` ;)

**Comment by igas** [comment]
Wow, it was fast! Great work @saulecabrera! Fix my comment and delete related json file, please. And I'll be glad to merge it.

**Comment by saulecabrera** [comment]
Hey @igas, I've pushed the requested changes!

**Comment by igas** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Comment by igas** [comment]
@saulecabrera if you have plans to do it with other modules please response in one of the [issues](https://github.com/igas/faker/labels/help%20wanted) to prevent double work.

**Pull request merged**